### PR TITLE
Merge dev changes

### PR DESCRIPTION
**Pull request by andrie**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+
+* Include `sortable.js` version 1.13.0, as suggested by #73
+
+Bug fixes
+
+* Capture error if bucket_list header is empty #31
+* Fix rank-list-item that spills outside the container boundary #68
+* Allow bucket_list to have empty header, and capture error better #69
+
+Other changes
```
Comment (schloerke) [inline NEWS.md:13]:
```suggestion
## Upgrade sortable.js

* Include `sortable.js` version 1.13.0, as suggested by #73

## Bug fixes

* Capture error if bucket_list header is empty #31
* Fix rank-list-item that spills outside the container boundary #68
* Allow bucket_list to have empty header, and capture error better #69

## Other changes
```

File: NAMESPACE [hunk 2]
```diff
@@ -31,4 +31,5 @@ importFrom(learnr,question_is_valid)
 importFrom(learnr,question_ui_completed)
 importFrom(learnr,question_ui_initialize)
 importFrom(learnr,question_ui_try_again)
+importFrom(rlang,list2)
```
Comment (schloerke) [inline NAMESPACE:34]:
```suggestion
```

File: R/bucket_list.R [hunk 3]
```diff
@@ -60,6 +60,8 @@ is_add_rank_list <- function(x) {
 #' @seealso rank_list
 #' @export
 #'
+#' @importFrom rlang list2
+#'
```
Comment (schloerke) [inline R/bucket_list.R:64]: No need to importFrom if using `::`
```suggestion
```

File: DESCRIPTION [hunk 4]
```diff
@@ -48,7 +49,8 @@ Suggests:
     magrittr,
     webshot,
     spelling,
-    covr
+    covr,
+    ragg
```
Comment (schloerke) [inline DESCRIPTION:53]: Why `ragg`? I don't see any usage

```suggestion
    covr
```

**Review by schloerke — COMMENTED** [review]
If `ragg` is removed and suggestions accepted, LGTM!

File: NEWS.md (hunk 1 above)
Comment (andrie) [reply NEWS.md:13]: Fixing.

File: NAMESPACE (hunk 2 above)
Comment (andrie) [reply NAMESPACE:34]: Fixing

File: DESCRIPTION (hunk 4 above)
Comment (andrie) [reply DESCRIPTION:53]: Good catch.  I don't know why this is in here.  Fixing

**Pull request merged**